views/layouts/default.tpl

```diff
@@ -5,16 +5,22 @@
   <title>Google Scraper</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  <link type="text/css" rel="stylesheet" href="static/css/application.css" />
-  <link type="text/css" rel="stylesheet" href="static/css/tailwind.css" />
+  {{ assets_css "/css/application.css" }}
```

Comment:
@olivierobert Yes `assets_css` is from `Beego`, more template functions can be found [here](https://beego.me/docs/mvc/view/template.md), also include tailwind on [this commit](https://github.com/carryall/go-google-scraper-challenge/pull/25/commits/a1568436e39115ac2bbce49d090e1f843a6a244a) 🙏🏼